src/mixins/filing-template-mixin.ts

```diff
@@ -377,7 +377,7 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     if (courtOrder.hasPlanOfArrangement || courtOrder.fileNumber) {
       filing.dissolution.courtOrder = {
         fileNumber: courtOrder.fileNumber,
-        effectOfOrder: courtOrder.hasPlanOfArrangement ? EffectOfOrders.PLAN_OF_ARRANGEMENT : null,
+        effectOfOrder: courtOrder.hasPlanOfArrangement ? EffectOfOrders.PLAN_OF_ARRANGEMENT : '',
```

Comment:
This would cause a failure when run against the schema as the schema calls for `string` type

Reply:
I think string allows null.

Reply:
🤔  It would resolve to `None` when being checked against the schema, so my assumption is that the backend would coerce `null` -> `None` type somewhere along the line. Potentially when saving the json, not 100% sure. 
Either way, this ensured we met schema requirements as it would fail on `effectOfOrder : None`
Think we should dig deeper to this?

Reply:
You may be right. I see this, which suggest that we have to explicitly allow null:

https://github.com/bcgov/business-schemas/blob/836b85359b67da49a2fc89c5aa6987806f896cef/src/registry_schemas/schemas/address.json#L34

Reply:
Ah right, i knew there was something previously that had been updated in order for the nulls to pass through. Thanks for asking/sharing this, this is very good to know!